spoilers.md

```diff
@@ -2,5 +2,5 @@
 
 ## car
 
-- Doesn't respond.
+- no attempt in formatting raw ass json formatting into a friendly ui.
```

Comment:
- Simply adds a field to the embed for every key in the object.  As a result field names use `camelCase` and fields that  are themselves objects, such as options, are displayed as raw JSON.

Reply:
That moment when you're reviewing pull requests at 6 AM and completely fail to add a suggestion as an actual suggestion.